packages/xrpl/src/models/methods/serverInfo.ts

```diff
@@ -33,6 +33,14 @@ export interface JobType {
   in_progress?: number
 }
 
+// The states for validating and proposing do not exist in the field state_accounting
+// See https://github.com/XRPLF/rippled/blob/develop/src/ripple/app/misc/NetworkOPs.cpp#L4545
+// https://github.com/XRPLF/rippled/blob/develop/src/ripple/app/misc/NetworkOPs.h#L66
+export type StateAccountingFinal = Record<
```

Comment:
I think the types would be more readable if we just used the `Record<..., StateAccounting>` type in the `ServerInfo` and `ServerState` models

Reply:
But those are still valid states for `ServerState` and `ServerInfo`, but not for `StateAccounting` fields?

Reply:
Sorry, I meant the whole thing - `Record<Exclude<ServerState, 'validating' | 'proposing'>, StateAccounting>`